### ISSUE TITLE
The keyboard was too high for interacting with the functionality

### ISSUE THREAD

**Issue by neilideal** [Yet-Zio/yetCalc#158]
Hey! Very cool app. Especially pleased with the added functionality in the latest update, but...

Something has become a bit inconvenient. In previous versions, the keyboard was at the level of my fingers, but now to press the keys requires reaching higher. Maybe consider reducing the bottom indentation to improve the user experience?

![Screenshot_20241201-202103_yetCalc.png](https://github.com/user-attachments/assets/c5c8ad85-f17f-421a-8041-672e2fe924fb)

**Comment by Yet-Zio** [comment]
Sure, I will try to fix the spacing for larger screens.

**Comment by neilideal** [comment]
Everything I noticed in this issue is now well documented in #154. Maybe this one should be closed so that there is no duplication of issues?

**Comment by Yet-Zio** [comment]
Nah, actually that issue is more concerned with the dynamic system font size. This one, however is because the layout is not chained to bottom.

**Comment by thecoldsea** [comment]
Can we get a compact layout option resembling traditional layout format (CA and <x| beside each-other) in contrast to default one? 

Perfect:
![buirgz1ub3bwjsyn jpg](https://github.com/user-attachments/assets/d825a1b4-55ad-4174-a9f7-202c9523f750)
